tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -40,6 +60,16 @@ def _store_test_result(self, name, actual, expected):
             cur_dict = cur_dict.setdefault(name_part, {})
         cur_dict["actual"] = actual
         cur_dict["expected"] = expected
+        if message != "":
+            cur_dict["artifacts"] = {"log": message}
+
+        # Figure out if there was a regression or unexpected status. This only
+        # happens for tests that were run
+        if actual != "SKIP":
+            if actual != expected:
+                cur_dict["is_unexpected"] = True
+                if actual != "PASS":
```

Comment:
How does this interact with testharness tests ("OK" instead of "PASS" on the test level)?

Reply:
All the statuses go through _map_status_name first, so we (hope to be) always working with canonical chromium statuses.